packages/maskbook/src/extension/background-script/SettingsService.ts

```diff
@@ -0,0 +1,16 @@
+import type { InternalSettings } from '../../settings/createSettings'
+import { appearanceSettings } from '../../settings/settings'
+
+function create<T>(settings: InternalSettings<T>) {
```

Comment:
Move this to settings? We could reuse it.

Reply:
This part is mainly for the new dashboard right now. The new dashboard cannot have direct access to the settings objects but code in the mask extension package can. They can directly read/write on the settings object.